<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>218</width>
    <height>391</height>
   </rect>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="mainLayout" stretch="1,0">
    <item>
     <widget class="TimersView" name="timersListView">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustToContents</enum>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::DoubleClicked</set>
      </property>
      <property name="showDropIndicator" stdset="0">
       <bool>false</bool>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="uniformItemSizes">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="detailsGroupBox">
      <property name="title">
       <string>Timer</string>
      </property>
      <layout class="QVBoxLayout" name="groupBoxLayout">
       <item>
        <widget class="QLabel" name="currentLabel">
         <property name="text">
          <string>Current</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="currentHorizontalLayout" stretch="1,0,1,0,1">
         <item>
          <widget class="QSpinBox" name="currentHoursSpinBox">
           <property name="minimum">
            <number>0</number>
           </property>
           <property name="maximum">
            <number>23</number>
           </property>
           <property name="value">
            <number>0</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="currentHoursLabel">
           <property name="text">
            <string>:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="currentMinutesSpinBox">
           <property name="maximum">
            <number>59</number>
           </property>
           <property name="singleStep">
            <number>10</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="currentMinutesLabel">
           <property name="text">
            <string>:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="currentSecondsSpinBox">
           <property name="maximum">
            <number>59</number>
           </property>
           <property name="singleStep">
            <number>10</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="defaultLabel">
         <property name="text">
          <string>Default</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="defaultHorizontalLayout" stretch="1,0,1,0,1">
         <item>
          <widget class="QSpinBox" name="defaultHoursSpinBox">
           <property name="minimum">
            <number>0</number>
           </property>
           <property name="maximum">
            <number>23</number>
           </property>
           <property name="value">
            <number>0</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="defaultHoursLabel">
           <property name="text">
            <string>:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="defaultMinutesSpinBox">
           <property name="maximum">
            <number>59</number>
           </property>
           <property name="singleStep">
            <number>10</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="defaultMinutesLabel">
           <property name="text">
            <string>:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="defaultSecondsSpinBox">
           <property name="maximum">
            <number>59</number>
           </property>
           <property name="singleStep">
            <number>10</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="nameLabel">
         <property name="text">
          <string>Name</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="nameEdit"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QCheckBox" name="absoluteTimeCheckBox">
           <property name="text">
            <string>Absolute time</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="colorButton">
           <property name="text">
            <string>Color</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="addButton">
           <property name="text">
            <string>Add</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="removeButton">
           <property name="text">
            <string>Remove</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>218</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionCheckUpdates"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
    <addaction name="actionAboutQt"/>
   </widget>
   <widget class="QMenu" name="menu_View">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="actionAlwaysOnTop"/>
    <addaction name="actionShowOnlyActiveTimers"/>
    <addaction name="actionShowDetails"/>
    <addaction name="actionShowMinimal"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menu_View"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionExit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
   <property name="menuRole">
    <enum>QAction::QuitRole</enum>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About...</string>
   </property>
   <property name="menuRole">
    <enum>QAction::AboutRole</enum>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="text">
    <string>About Qt</string>
   </property>
   <property name="menuRole">
    <enum>QAction::AboutQtRole</enum>
   </property>
  </action>
  <action name="actionCheckUpdates">
   <property name="text">
    <string>Check updates...</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>&amp;Open...</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>&amp;Save</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="text">
    <string>Save &amp;as...</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>&amp;New</string>
   </property>
  </action>
  <action name="actionAlwaysOnTop">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Always on top</string>
   </property>
  </action>
  <action name="actionShowOnlyActiveTimers">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show only active timers</string>
   </property>
  </action>
  <action name="actionShowDetails">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show details</string>
   </property>
  </action>
  <action name="actionShowMinimal">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show minimal</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TimersView</class>
   <extends>QListView</extends>
   <header>timersview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
